### ISSUE TITLE
Question about the output of the Unet

### ISSUE THREAD

**Issue by lixiaoqingnnz** [hkproj/pytorch-stable-diffusion#3]
Hi, thank you so much for your work and sharing.

I want to retrain the Unet from the beginning, may I ask what is the output of the Unet according to this ddpm scheduler? Whether it is the random noise or the noise computed with the specific time step? I saw other codes, they use the random noise as the output of the model, but my model only works when I used the later one. 

Highly appreciate your help!

**Comment by hkproj** [comment]
Hi! The input of the Unet is `image + noise + timestep`, the output of the Unet is `noise` (how much noise was added to the input).
If you want to train your own Unet, I recommend training a Diffusion Model, and not a Latent Difusion Model. Please check my implementation of the DDPM paper at: https://github.com/hkproj/pytorch-ddpm
This implementation also has a training code.
